cli-plugins/apps/reference/clustersupplychain.hbs.md

```diff
@@ -2,16 +2,17 @@
 
 This command provides details about the cluster supply chain.
 
-## Tanzu apps cluster supply chain list
+## tanzu apps clustersupplychain list
 
 The `tanzu apps clustersupplychain list` command lists the available supply chains installed in the
-cluster (supported clustersupplychain alias is `csc`).
+cluster (supported clustersupplychain aliases are `csc`, `cluster-supply-chain`, 
```

Comment:
remove the aliases content here...provide the aliases for this command further above, since they apply to all sub-commands.

Reply:
ok, removed